src/main/java/hw_3/Task3.java

```diff
@@ -0,0 +1,22 @@
+package hw_3;
+
+public class Task3 {
+    public static void main(String[] args) {
+        // ДЗ Масиви та рядки
+        /*Створіть рядок "Hello world!"
+        - Виведіть перший символ
+        - Виведіть останній символ
+        - Створіть масив символів, покладіть туди усі символи слова world
+        - Вивести масив у консоль
+        - Вивести в консоль передостанній елемент масиву, не використовуючи прямий доступ за індексом (arr[3] – не згодиться)
+        */
+        String hello = "Hello World!";
+        char first = hello.charAt(0);
+        char last = hello.charAt(hello.length() - 1);
+        System.out.println(first);
+        System.out.println(last);
+        char[] world = {'w','o','r','l','d'};
```

Comment:
You need do get a 'world' word from the "Hello world!" string and then you need to put all symbols from 'world' word into array.
For now, you just create an array and put there symbols. You need to practise string methods

Reply:
fixed